Le_duong/LoadDB/connect.php

```diff
@@ -0,0 +1,52 @@
+<?php
+	 //Connect server 
+	$servername="localhost";
+	$username="leduong";
+	$password="Leduong.4dec";
+	$dbname="php2104";
```

Comment:
```suggestion
	$servername = '***';
	$username = '***';
	$password = '***';
	$dbname = '***';
```
thông tin db để như này thôi nhé, kẻo sau server thật lại lộ hết thông tin
- Trước và sau dấu `=` có 1 dấu cách
- Nên dùng `'` thay cho `"`